client/src/components/Mentions.jsx

```diff
@@ -4,28 +4,40 @@ import styles from '../styles/Mentions.css';
 class Mentions extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      expanded: false,
+    };
+  }
+
+  handleSeeMoreClick() {
+    const newExpanded = !this.state.expanded;
+    this.setState({
+      expanded: newExpanded,
+    });
   }
 
   render() {
-    const state = this.props.getState();
-    const { reviews } = state;
+    const mentionPhrases = ['lorem', 'ipsum', 'is', 'simply', 'dummy', 'text', 'of', 'the', 'printing', 'and', 'typesetting', 'industry', 'lorem', 'ipsum', 'is', 'simply', 'dummy', 'text', 'of', 'the', 'printing', 'and', 'typesetting', 'industry'];
```

Comment:
Is every product going to have these mentionPhrases?

Reply:
at the moment yes, this is another feature that is a reach goal